apple/Sources/Sargon/Extensions/Swiftified/Profile/Profile+Swiftified.swift

```diff
@@ -1,7 +1,19 @@
 import SargonUniFFI
 
 extension Profile: SargonModel {}
-extension Profile: SargonObjectCodable {}
+
+@available(*, unavailable, message: "Profile should not use Swift `Encodable` (Codable), since it is too slow.")
+extension Profile: Encodable {
+	@available(*, unavailable, message: "Profile should not use Swift `Encodable` (Codable), since it is too slow.")
+	public func encode(to encoder: any Encoder) throws { fatalError("Unreachable") }
```

Comment:
CodeCov will complain about these lines, and they are.... completely untestable.